ts-nestjs-sequelize/.gitignore

```diff
@@ -0,0 +1,35 @@
+# compiled output
+/dist
+/node_modules
+
+# Logs
```

Comment:
for pretty much all of these (not just this section, but everything else as well), I feel like they could go in the repo's root gitignore. That way we won't have to maintain one for each new sample app.